UnitySDK/Assets/ML-Agents/Scripts/InferenceBrain/ModelParamLoader.cs

```diff
@@ -226,12 +226,12 @@ private static ModelActionType GetActionType(int isContinuousInt)
         /// <param name="requiredScalarFields"> Mapping from node names to int values</param>
         private void CheckIntScalarPresenceHelper(Dictionary<string, int> requiredScalarFields)
         {
-            foreach(var field in requiredScalarFields)
-            if (field.Value == -1)
-            {
-                _failedModelChecks.Add(
-                    $"Missing node in the model provided : {field.Key}");
-            }
+            foreach (var field in requiredScalarFields)
+                if (field.Value == -1)
```

Comment:
I think this needs `{}` (if I'm reading the guide correctly).

Reply:
this is true, yet this whole class is `#if`ed out of existence.  I'll add the braces.